src/software/js-samples/importDevices.html

```diff
@@ -124,7 +124,10 @@ <h2>Help</h2>
                                 alert("Group name on line " + (i + 1) + " exceeds 255 characters");
                                 return;
                             }
-
+                            if (split[3].length > 255) {
```

Comment:
@dab09 I think that this might still be a split[2] for checking the length of the GroupName, with the split[3] it's evaluating the VIN.